src/System.Collections/tests/Generic/HashSet/HashSet.Generic.Tests.cs

```diff
@@ -162,6 +162,122 @@ public void HashSet_Generic_RemoveWhere_NullMatchPredicate(int setLength)
 
         #endregion
 
+        #region EnsureCapacity
```

Comment:
I'd have expected these tests would need to be added to HashSet.Generic.Tests.netcoreapp.cs instead; does CI pass with these here?

Reply:
No, NETX is failing, they need to be moved. Good catch.